src/components/common/YourCompany/StartDate.vue

```diff
@@ -0,0 +1,214 @@
+<template>
+  <section id="business-start-date">
+    <v-row no-gutters>
+      <v-col cols="3" class="pr-2">
+        <label :class="{'error-text': invalidSection}">
+          <strong>Business Start Date</strong>
+        </label>
+        <v-chip
+          v-if="isCorrected"
+          x-small label
+          color="primary"
+          text-color="white"
+        >
+          <span>{{ editedLabel }}</span>
+        </v-chip>
+      </v-col>
+
+      <!-- Info tooltip-->
+      <v-col cols="9">
+        <span v-if="!onEditMode" class="info-text mr-1">{{ recognitionDateTime }}</span>
+        <v-tooltip v-if="!isCorrectionFiling"
+                    top
+                    content-class="top-tooltip"
+                    transition="fade-transition"
+                    nudge-right="3"
+        >
+          <template v-slot:activator="{ on }">
+            <v-icon v-on="on" class="info-icon">mdi-information-outline</v-icon>
+          </template>
+          <span>If the business start date is incorrect, it must be corrected through a correction filing.</span>
```

Comment:
Isn't THIS a correction filing? Maybe we don't need the tooltip here. Check with Yui.

Reply:
Yes but this component is still used in change filings as well

Reply:
You are right we do, I will verify with Yui

Reply:
Oh right -- the tooltip is only displayed for "non correction" filings.

Actually it's bad practice to use "not something" here. It would be easier to understand (and extend) if this used "is something" (eg, is alteration) instead.

Reply:
Okay I will change that to a verify that it is for something here instead of checking a negation

Reply:
Don't resolve the conversations that I started (unless I ask you to :) ).

Reply:
This looks OK.